dap_aria_mapping/pipeline/entity_processing/reformat_entities.py

```diff
@@ -0,0 +1,87 @@
+from typing import Dict
+import boto3
+import json
+
+from metaflow import FlowSpec, step, Parameter, batch
+
+
+PATENT_ANNOTATION_OUTPUT = "inputs/data_collection/patents/preprocessed_annotated_patents.json"
+OPENALEX_ANNOTATION_OUTPUT = "inputs/data_collection/processed_openalex/preprocessed_annotated_abstracts.json"
+INPUTS = [
+    "inputs/data_collection/patents/annotated_patents.json",
+    "inputs/data_collection/processed_openalex/annotated_abstracts.json"
+]
+
+
+def reformat_dictionary(dictionary: Dict) -> Dict:
```

Comment:
We could still use this flow and just modify slightly to accomodate the new format i.e.  ```
'WO-2020254804-A1': [{'URI': 'http://dbpedia.org/resource/The',
   'confidence': 30},
  {'URI': 'http://dbpedia.org/resource/RNA', 'confidence': 90},
  {'URI': 'http://dbpedia.org/resource/Social_constructionism',
   'confidence': 30}]
```
to get rid of ents with confidence less than 70 and to strip the URI business!